src/index.html

```diff
@@ -59,66 +61,106 @@ <h1>Wherever you need to go, we'll keep your pooch safe, and happy.</h1>
-        <div class="o-column-33">Single Col</div>
-        <div class="o-column-33">Single Col</div>
-      </div>
-
-    </div>
-    <div class="o-section c-section__remote">
-      <!-- Remote Access -->
-      <h2>Remote Access</h2>
-      <hr class="c-section--hr-thin">
-    </div>
-  </div>
+      </article>
+    </section>
+  </main>
+
+  <!-- Our Locations -->
+  <section class="c-feature">
+    <header class="c-feature__header">
+        <h2 class="o-underline">Our Locations</h2>
+    </header>
+    
+    <ul class="c-feature__list">
+      <li class="c-feature__item">
+        <img class="c-feature__image" src="./assets/Map.png" alt="Map of Avondale Location">
+        <h3>Chicago - Avondale</h3>
+        <p>Our original location located in the dog-friendly neighborhood of Avondale. Easy drop-off and pick-up.</p>
+      </li>
+      <li class="c-feature__item">
+        <img class="c-feature__image" src="./assets/Map.png" alt="Map of Portage Park Location">
+        <h3>Chicago - Portage Park</h3>
+        <p>Our newest location with increased capacity, same great quality of services, and a pool!</p>
+      </li>
+    </ul>
+  </section>
+
+  <!--  New Features -->
+  <section class="c-feature">
+    <header class="c-feature__header o-underline">
+      <h2>What's great about staying with us?</h2>
+      <p>Still on the fence?</p>
+    </header>
+
+    <ul class="c-feature__list">
+      <li class="c-feature__item c-feature__item--third">
+        <img class="c-feature__image" src="./assets/Map.png" alt="Dog in backseat of car">
+        <h3>It's convenient</h3>
+        <p>
+          Whether you drop off your pet, or use our pick-up service, we're available when you need it.
+        </p>
+        <a href="somewhere.html" class="c-feature__link">
+          Our Hours
+        </a>
+      </li>
+
+      <li class="c-feature__item c-feature__item--third">
+        <img class="c-feature__image" src="./assets/Map.png" alt="Dog in backseat of car">
+        <h3>It's convenient</h3>
+        <p>
+          Whether you drop off your pet, or use our pick-up service, we're available when you need it.
+        </p>
+        <a href="somewhere.html" class="c-feature__link">
+          Our Hours
+        </a>
+      </li>
+
+      <li class="c-feature__item c-feature__item--third">
+        <img class="c-feature__image" src="./assets/Map.png" alt="Dog in backseat of car">
+        <h3>It's convenient</h3>
+        <p>
+          Whether you drop off your pet, or use our pick-up service, we're available when you need it.
+        </p>
+        <a href="somewhere.html" class="c-feature__link">
+          Our Hours
+        </a>
+      </li>
+    </ul>
+  </section>
+
+  <!-- Remote Access -->
+  <section class="o-section c-section__remote">
+    <header class="c-feature__header">
+      <h2 class="o-underline">Remote Access</h2>
+    </header>
+
+  </section>
+
   <footer>
     <!-- 4 col footer -->
-    <h2>It'sa spicy footer</h2>
```

Comment:
I prefer this old copy 

(this is not real feedback)

Reply:
🔥🥩🏀